server/src/main/java/org/apache/iotdb/db/metadata/mnode/MeasurementMNode.java

```diff
@@ -205,4 +220,88 @@ public TSDataType getDataType(String measurementId) {
       return schema.getValueTSDataTypeList().get(index);
     }
   }
+
+  @Override
+  public boolean hasChild(String name) {
+    return false;
+  }
+
+  @Override
+  public void addChild(String name, IMNode child) {
+    // Do nothing
+  }
+
+  @Override
+  public IMNode addChild(IMNode child) {
+    return null;
+  }
+
+  @Override
+  public void deleteChild(String name) {
+    // Do nothing
+  }
+
+  @Override
+  public void deleteAliasChild(String alias) {
+    // Do nothing
+  }
+
+  @Override
+  public Template getDeviceTemplate() {
+    return null;
```

Comment:
May be throw a exception here and explain we can't get template from a measurement node

Reply:
Fixed